third-party/libunwind/Makefile.include

```diff
@@ -0,0 +1,11 @@
+LIBUNWIND_DIR=$(THIRD_PARTY_DIR)/libunwind
+LIBUNWIND_SRC_DIR=libunwind-1.1
```

Comment:
We are moving toward a structure in which the third-party package version number is not part of the path. An  example is for GASNet, where the tarball contents are placed in $CHPL_HOME/third-party/gasnet/gasnet-src/ instead of (for example) .../GASNet-1.26.3/. Could you set up third-party/libunwind the same way? third-party/gasnet/README might also make a good template for third-party/libunwind/README. You'll have to adjust the last section of it because for now we are expecting users to download and untar libunwind themselves, but that should be a simple change.